### DISCUSSION TITLE
Support Excalidraw Embeds

### DISCUSSION THREAD

**Discussion by kadisonm** [alangrainger/share-note#46] [Vote for upcoming features]
Support the excalidraw’s embed within notes. As of current, the embed is ignored in the shared link.

**Comment by kwilcz** [comment]
Excalidraw can auto-generate svg images on save. You can then easily embed those.

**Comment by alangrainger** [comment]
After much trawling through the Excalidraw plugin code, I have figured out how to fetch and share Excalidraw drawings :)

Live in 1.3.0:

https://share.note.sx/437mhqbk#g6MKOQHMWkKgbpKrdCpT7GLk4RgAIhIIlKVie+vyJiM
